cli/check/main.rs

```diff
@@ -0,0 +1,102 @@
+extern crate parity_wasm;
+extern crate pwasm_utils as utils;
+extern crate pwasm_utils_cli as logger;
+extern crate clap;
+
+use clap::{App, Arg};
+use parity_wasm::elements;
+
+fn fail(msg: &str) -> ! {
+    eprintln!("{}", msg);
+    std::process::exit(1)
+}
+
+const ALLOWED_IMPORTS: &'static [&'static str] = &[
+    "ret",
+    "storage_read",
+    "storage_write",
+    "balance",
+    "sender",
+    "origin",
+    "fetch_input",
+    "input_length",
+    "ccall",
+    "dcall",
+    "scall",
+    "create",
+    "balance",
+    "blockhash",
+    "blocknumber",
+    "coinbase",
+    "timestamp",
+    "difficulty",
+    "gaslimit",
+    "address",
+    "value",
+    "suicide",
+    "panic",
+    "elog",
+    "abort"
```

Comment:
so you can't use `gas`, can you? : )

Reply:
sure, because it supposed to check compiled contracts, which should not import gas manually